src/ep.config.json

```diff
@@ -1,8 +1,8 @@
 {
   "cortexApi": {
     "path": "/cortex",
-    "scope": "vestri",
-    "pathForProxy": "http://localhost:9080",
+    "scope": "mobee",
```

Comment:
You need to revert the changes in this file so they don't get merged. The local config shouldn't be pushed

Reply:
ok. forgot this) I will revert the changes